### PR TITLE
Hyperlink to support period update

### PR DESCRIPTION
**Pull request by shivam-sharma7**
- Hyperlink to support period update

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
@xmudrii

File: content/en/releases/_index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ type: docs
 
 <!-- overview -->
 
-The Kubernetes project maintains release branches for the most recent three minor releases ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}).  Kubernetes 1.19 and newer receive approximately 1 year of patch support. Kubernetes 1.18 and older received approximately 9 months of patch support.
+The Kubernetes project maintains release branches for the most recent three minor releases ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}).  Kubernetes 1.19 and newer receive [approximately 1 year of patch support](https://kubernetes.io/releases/patch-releases/#support-period). Kubernetes 1.18 and older received approximately 9 months of patch support.
```
Comment (jihoon-seo) [inline content/en/releases/_index.md:10]: I suggest to remove hardcoded site domain name:

```suggestion
The Kubernetes project maintains release branches for the most recent three minor releases ({{< skew latestVersion >}}, {{< skew prevMinorVersion >}}, {{< skew oldestMinorVersion >}}).  Kubernetes 1.19 and newer receive [approximately 1 year of patch support](/releases/patch-releases/#support-period). Kubernetes 1.18 and older received approximately 9 months of patch support.
```

**Review by jihoon-seo — COMMENTED** [review]
@shivam-sharma7 PTAL!

**Comment by jihoon-seo** [comment]
/label tide/merge-method-squash

This PR covers part of #32930
(from https://github.com/kubernetes/website/pull/32945#issuecomment-1100644227)

**Pull request merged**